Sources/Basic/TerminalController.swift

```diff
@@ -80,9 +80,11 @@ public final class TerminalController {
 
         // Try determining using ioctl.
         var ws = winsize()
+#if !CYGWIN
```

Comment:
I couldn't use it.
`os(Cygwin)` PR (https://github.com/apple/swift/pull/2351) was not accepted for Swift compiler.
There were some opinions that Cygwin is only an environment and not OS, and `os(Windows)` is true for all Windows variants.
We can not distinguish the Cygwin from other Windows variant, MSVC, MinGW.
`os(Windows)` and CYGWIN macro will be evaluated to true for Cygwin environment.
The CYGWIN macro is also used in Swift standard library.